api/requirements.txt

```diff
@@ -34,7 +34,14 @@ aioredis>=1.3.1,<1.4.0
 fsspec==2022.8.2
 s3fs==2022.8.2
 torch==1.13.0
-sentence-transformers==2.2.2
+# sentence-transformers==2.2.2
 transformers==4.26.1
 aiofiles==22.1.0
 openpyxl==3.0.7
+openai[embeddings]==0.27.10
+numpy==1.22
+sse-starlette==1.8.2
```

Comment:
This was added to stream the knowledge ai endpoint responses. Not particularly in use right now, but backend supports it.